src/codeu/chat/util/Uuid.java

```diff
@@ -168,7 +168,7 @@ private static int hash(Uuid id) {
   private static String toString(Uuid id) {
     final StringBuilder build = new StringBuilder();
     buildString(id, build);
-    return String.format("[UUID:%s]", build.substring(1));  // index of 1 to skip initial '.'
```

Comment:
If we are printing the UUID somewhere in the code, it would make sense to prepend "[UUID]:" there, now that it's removed from here.